test/e2e/apiserver.go

```diff
@@ -37,24 +37,27 @@ import (
 )
 
 const (
-	compliantImage1     = "registry.opensource.zalan.do/teapot/skipper:v0.14.0" // these are several compliant images
-	compliantImage2     = "registry.opensource.zalan.do/teapot/skipper:v0.14.1"
-	compliantImage3     = "registry.opensource.zalan.do/teapot/skipper:v0.14.2"
-	compliantImage4     = "registry.opensource.zalan.do/teapot/skipper:v0.14.3"
-	compliantImage5     = "registry.opensource.zalan.do/teapot/skipper:v0.14.4"
-	compliantImage6     = "registry.opensource.zalan.do/teapot/skipper:v0.14.5"
-	compliantImage7     = "registry.opensource.zalan.do/teapot/skipper:v0.14.6"
-	compliantImage8     = "registry.opensource.zalan.do/teapot/skipper:v0.14.7"
-	nonCompliantImage1  = "registry.opensource.zalan.do/teapot/skipper-test:pr-2080-2" // these are several non-compliant images
-	nonCompliantImage2  = "registry.opensource.zalan.do/teapot/skipper-test:pr-2080-3"
-	nonCompliantImage3  = "registry.opensource.zalan.do/teapot/skipper-test:pr-2080-5"
-	nonCompliantImage4  = "registry.opensource.zalan.do/teapot/skipper-test:pr-2080-6"
-	nonCompliantImage5  = "registry.opensource.zalan.do/teapot/skipper-test:pr-2080-7"
-	nonCompliantImage6  = "registry.opensource.zalan.do/teapot/skipper-test:pr-2080-8"
-	nonCompliantImage7  = "registry.opensource.zalan.do/teapot/skipper-test:pr-2080-10"
-	nonCompliantImage8  = "registry.opensource.zalan.do/teapot/skipper-test:pr-2080-11"
-	nonCompliantImage9  = "registry.opensource.zalan.do/teapot/skipper-test:pr-2080-12"
-	nonCompliantImage10 = "registry.opensource.zalan.do/teapot/skipper-test:pr-2080-13"
+	// these are several compliant images
+	compliantImage1 = "container-registry.zalando.net/teapot/skipper:v0.21.0"
+	compliantImage2 = "container-registry.zalando.net/teapot/skipper:v0.21.1"
+	compliantImage3 = "container-registry.zalando.net/teapot/skipper:v0.21.2"
+	compliantImage4 = "container-registry.zalando.net/teapot/skipper:v0.21.3"
+	compliantImage5 = "container-registry.zalando.net/teapot/skipper:v0.21.4"
+	compliantImage6 = "container-registry.zalando.net/teapot/skipper:v0.21.5"
+	compliantImage7 = "container-registry.zalando.net/teapot/skipper:v0.21.6"
+	compliantImage8 = "container-registry.zalando.net/teapot/skipper:v0.21.7"
```

Comment:
These must be real images (i.e. the tag must exist and the image must be considered compliant or non-compliant (see images below) by docker-meta). The reason why there's so many test images is to ensure that the tests using them aren't affected by any caching because that can lead to wrong results.

Meaning we need 8 compliant and 10 non-compliant images for the test cases or refactor the test cases.

Reply:
I can look into this today, thanks for the review 🙏